### PR TITLE
chore(repo): further-updates-to-ui-workflows

### PR DESCRIPTION
**Pull request by kimo-ice**
Handle reusable workflows limitations with secrets and steps.

### CODE REVIEW

**Review by KorbinianK — APPROVED** [review]

File: .github/workflows/bridge-ui.yml [hunk 1]
```diff
@@ -12,58 +12,35 @@ on:
...
-      - name: Checkout repository
-        uses: actions/checkout@v4
-      - name: Build and Checks
-        uses: ./.github/workflows/bridge-ui-ci.yml
+    uses: ./.github/workflows/bridge-ui-ci.yml
 
   # Deployment name follow the pattern: deploy_<appname(bridge-ui)>_<network(devnet|hekla|mainnet)>_<environment(preview|production)>
   deploy_bridge-ui_hekla_preview:
     if: ${{ github.ref_name != 'main' }}
-    runs-on: [taiko-runner]
```
Comment (d1onys1us) [inline .github/workflows/bridge-ui.yml:25]: is this not needed or is it intentional? this is our heavier runner

File: .github/workflows/taikoon-ui.yml [hunk 2]
```diff
@@ -12,43 +12,25 @@ on:
...
-    runs-on: [taiko-runner]
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-      - name: Build and Checks
-        uses: ./.github/workflows/taikoon-ui-ci.yml
+    uses: ./.github/workflows/taikoon-ui-ci.yml
 
-  # Deployment name follow the pattern: deploy_<appname(taikoon-ui)>_<network(devnet|hekla|mainnet)>_<environment(preview|production)>
+  # Deployment name follow the pattern: deploy_<appname(taikoon-ui)>_<network(devnet|mainnet|mainnet)>_<environment(preview|production)>
```
Comment (d1onys1us) [inline .github/workflows/taikoon-ui.yml:17]:
```suggestion
  # Deployment name follow the pattern: deploy_<appname(taikoon-ui)>_<network(devnet|hekla|mainnet)>_<environment(preview|production)>
```

File: .github/workflows/bridge-ui.yml (hunk 1 above)
Comment (kimo-ice) [reply .github/workflows/bridge-ui.yml:25]: Yes, intentional. With reusable workflows we can just reference the target workflow without specifying a runner as we actually have no runnable steps in the job.

**Review by dionysuzx — APPROVED** [review]
LGTM

**Pull request merged**